src/main/java/net/openhft/chronicle/bytes/internal/SingleMappedFile.java

```diff
@@ -120,7 +120,7 @@ private void resizeRafIfTooSmall(@NonNegative final long minSize)
             return;
 
         // handle a possible race condition between processes.
-        try {
+        try (final ReentrantThreadScopedFileLock lock = ReentrantThreadScopedFileLock.lock(file(), fileChannel)) {
```

Comment:
If you've acquired this lock you know you're the only one on the machine with it.